tarfs_test.go

```diff
@@ -18,11 +18,43 @@ func TestOpen(t *testing.T) {
 
 	// Add some files to the archive.
 	var files = []struct {
-		Name, Body string
+		Name        string
+		Body        string
+		AccessNames []string
 	}{
-		{"readme.txt", "This archive contains some text files."},
-		{"gopher.txt", "Gopher names:\nGeorge\nGeoffrey\nGonzo"},
-		{"todo.txt", "Get animal handling licence."},
+		{
+			Name: "readme.txt",
+			Body: "This archive contains some text files.",
+			AccessNames: []string{
+				"readme.txt",
+				"/readme.txt",
+				"./readme.txt",
+				"././readme.txt",
+				"../readme.txt",
```

Comment:
I am not sure about this. Isn't `../readme.txt` different from `./readme.txt`? The former one being the parent directory rather than current.

Reply:
> Isn't `../readme.txt` different from`./readme.txt`?

In most circumstances, but not at the filesystem root.  See 4 [here][1].

[1]: https://golang.org/pkg/path/#Clean

Reply:
Makes sense.

Reply:
That is pretty weird, but I suppose it is better to be consist with the standard library, so it all looks good.